src/components/FlowRunsAccordionContent.vue

```diff
@@ -24,20 +23,22 @@
 
   const limit = computed(() => props.flowRunLimit ?? 3)
 
-  const filter = computed<FlowRunsFilter>(() => ({
+  const filter = (): FlowRunsFilter => ({
```

Comment:
Is the refactors from computed getters to just getters for simplicity and/or perf? otherwise why?

Reply:
Correct less code and more performant. Originally our compositions accepted `MaybeRef` but since vue introduced the `MaybeRefOrGetter` and `toValue` I've been trying to update all the compositions to accept getters and then remove unnecessary computed properties like this.